DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/MapViewModel.cs

```diff
@@ -35,13 +38,54 @@ public MapViewModel() : base(Settings.UrlBase)
                         };
         }
 
-        private async Task ExecuteLoadRoutesCommand()
+        private async Task ExecuteLoadDataCommand()
         {
             if (IsBusy)
                 return;
 
             IsBusy = true;
 
+            try
+            {
+                await ExecuteLoadBuses();
```

Comment:
I think you can use Task.WhenAll() here to run both loads at once. Currently it waits to load buses before it starts trying to load routes.